data/update.py

```diff
@@ -108,38 +112,83 @@ def run(options):
   if options.get("upload", False):
     print("[%s] Syncing scan data and database to S3." % the_date)
     print()
-    upload(the_date)
+    upload_s3(the_date)
     print()
     print("[%s] Scan data and database now in S3." % the_date)
 
   print("[%s] All done." % the_date)
 
 
 # Upload the scan + processed data to /live/ and /archive/ locations by date.
-def upload(date):
-  live_parents = "s3://%s/live/parents/" % (BUCKET_NAME)
-  live_subdomains = "s3://%s/live/subdomains/" % (BUCKET_NAME)
-  live_db = "s3://%s/live/db/" % (BUCKET_NAME)
-  archive_subdomains = "s3://%s/archive/%s/subdomains/" % (BUCKET_NAME, date)
-  archive_parents = "s3://%s/archive/%s/parents/" % (BUCKET_NAME, date)
-  archive_db = "s3://%s/archive/%s/db/" % (BUCKET_NAME, date)
-
+def upload_s3(date):
+  # Used for all operations.
   acl = "--acl=public-read"
 
-  shell_out(["aws", "s3", "sync", PARENTS_DATA, live_scanned, acl])
-  shell_out(["aws", "s3", "sync", SUBDOMAIN_DATA, live_subdomains, acl])
+  # Used when uploading to the live/ dir.
+  delete = "--delete"
```

Comment:
Note that the `aws s3 sync` command is now using `--delete` when uploading a new scan to S3. We weren't using this before, which meant that old data was getting copied into new scans. This wouldn't impact the results, but would mean that the `cache/` dir was growing steadily. There were `cache/inspect/` directories in the latest scan, even though we moved from `site-inspector` to `pshtt` over a year ago.